packages/schemas/src/db-entries/user.ts

```diff
@@ -55,6 +59,7 @@ export const Users: GeneratedSchema<CreateUser> = Object.freeze({
     'passwordEncrypted',
     'passwordEncryptionMethod',
     'passwordEncryptionSalt',
+    'userRoles',
```

Comment:
just roles？

Reply:
would conflict with the roles table name 🤣

Reply:
<img width="644" alt="image" src="https://user-images.githubusercontent.com/36393111/151146058-03dca831-b691-491e-a091-3b7cea27fdaf.png">

Reply:
feel free to shoot any suggestions

Reply:
aligned offline userRoles -> roleNames